fairing/utils.py

```diff
@@ -33,3 +35,51 @@ def crc(file_name):
 
 def random_tag():
     return str(uuid.uuid4()).split('-')[0]
+
+
+def guess_preprocessor(entry_point, *args, **kwargs):
+    #Trying to avoid any clyclic dependencies by importing locally within the method
+    from fairing.functions.function_shim import get_execution_obj_type, ObjectType
```

Comment:
this should be an indicator that utils is not the right package for this

Reply:
Ack.